scripts/zulip_emoji_merge_delegate.py

```diff
@@ -102,6 +103,13 @@
                 "emoji_name": "writing"
             })
             print(f"result: '{result}'")
+        if has_closed:
+            print('Removing awaiting-author')
```

Comment:
The comment is wrong (copy-pasta) and the code right, correct?

Reply:
Oops, thanks!